### PR TITLE
feature/SOF-7492 clusters NBs

### PR DESCRIPTION
**Pull request by VsevolodX**
- feat: add sphere cluster notebook
- feat: add nb for box cutout
- chore: small fix
- update: add links to intro nb
- chore: run lint fix

### CODE REVIEW

File: other/materials_designer/Introduction.ipynb [hunk 1]
```diff
@@ -26,6 +26,8 @@
     "\n",
```
Comment (timurbazhirov) [inline other/materials_designer/Introduction.ipynb:26]: <p>remove "transistor by default"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/163/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

File: other/materials_designer/create_box_cutout.ipynb [hunk 2]
```diff
@@ -0,0 +1,283 @@
+{
```
Comment (timurbazhirov) [inline other/materials_designer/create_box_cutout.ipynb:1]: <p>Can we add an image to demonstrate what MAX, MIN_COORDINATES are</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/163/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

File: other/materials_designer/create_cluster_sphere.ipynb [hunk 3]
```diff
@@ -0,0 +1,255 @@
+{
```
Comment (timurbazhirov) [inline other/materials_designer/create_cluster_sphere.ipynb:1]: <p>Create a cluster from a bulk material</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/163/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (timurbazhirov) [inline other/materials_designer/create_cluster_sphere.ipynb:1]: <p>Let's add a notebook for a custom 3D shape cluster.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/163/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (timurbazhirov) [inline other/materials_designer/create_cluster_sphere.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #8.    
</pre><p>Remove newline</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/163/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (timurbazhirov) [inline other/materials_designer/create_cluster_sphere.ipynb:1]: <p>We should change the lattice to simple cubic with a specific value of vacuum padding</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/163/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (timurbazhirov) [inline other/materials_designer/create_cluster_sphere.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #5.    DEFAULT_SLAB_PARAMETERS = {
</pre><p>Let's explain above that we align the Z axis of the cluster with a direction defined by the miller indices.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/163/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

File: other/materials_designer/create_custom_shape.ipynb [hunk 4]
```diff
@@ -0,0 +1,336 @@
+{
```
Comment (timurbazhirov) [inline other/materials_designer/create_custom_shape.ipynb:1]: <p>create_custom_cutout - for the file name</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/163/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (timurbazhirov) [inline other/materials_designer/create_custom_shape.ipynb:1]: <p>Let's explain what's happening below - 80 lines of code</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/163/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (timurbazhirov) [inline other/materials_designer/create_custom_shape.ipynb:1]: <p>Let's add an image of the final result to showcase the final result</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/163/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

File: other/materials_designer/create_cluster_sphere.ipynb (hunk 3 above)
Comment (VsevolodX) [reply other/materials_designer/create_cluster_sphere.ipynb:1]: <p>It's a formatter working. Previously we didn't run it. </p>

File: other/materials_designer/create_custom_shape.ipynb (hunk 4 above)
Comment (VsevolodX) [reply other/materials_designer/create_custom_shape.ipynb:1]: Actually, there is explanation at the very begiining of the notebook. What we're building

File: other/materials_designer/create_cutout_box.ipynb [hunk 5]
```diff
@@ -0,0 +1,376 @@
+{
```
Comment (timurbazhirov) [inline other/materials_designer/create_cutout_box.ipynb:1]: <p>What is this code - is this for a plot? Save in "utils"/"scripts" and call appropriately</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/163/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

File: other/materials_designer/create_cutout_custom_shape.ipynb [hunk 6]
```diff
@@ -0,0 +1,341 @@
+{
```
Comment (timurbazhirov) [inline other/materials_designer/create_cutout_custom_shape.ipynb:1]: <p>Create a custom-shaped cutout</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/163/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

**Review by timurbazhirov — APPROVED** [review]

File: other/materials_designer/create_cutout_box.ipynb (hunk 5 above)
Comment (VsevolodX) [reply other/materials_designer/create_cutout_box.ipynb:1]: <p>I think it's a leftover from creation of that cube graphic. Commited by an error. </p>

File: other/materials_designer/create_cutout_custom_shape.ipynb (hunk 6 above)
Comment (VsevolodX) [reply other/materials_designer/create_cutout_custom_shape.ipynb:1]: <p>Ok</p>

**Pull request merged**